azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/HandlerFactoryImpl.java

```diff
@@ -49,7 +49,7 @@ public RuntimeHandler getRuntimeHandler(final WebAppConfiguration config, final
         }
         return builder.appName(config.getAppName())
             .resourceGroup(config.getResourceGroup())
-            .region(config.getRegion())

```

Comment:
Change to `Region` in next PR

Reply:
Sure, the change involves util function API change and unit tests change, will complete it in next PR and attach the link here.